mobify/platform-scaffold
# WEB-1230: Implement log out functionality

**JIRA**: https://mobify.atlassian.net/browse/WEB-1230

## Changes
- Update local nav-item component to accept custom `onClick` behaviour
- Add `signOut` action which will display notification, update cart items and update the navigation menu

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Log into an account
- Navigate to Potions PLP
- Open menu and click on sign out
- Reopen menu and you should see sign in link again while Potions item is still highlighted
